### ISSUE TITLE
Notification e-mail templates

### ISSUE THREAD

**Issue by dwlhs** [dynamicweb/DynamicWeb#87]
Hi :)

The data integration templates for notification emails are not included in DW10 nor Swift:

![image](https://github.com/dynamicweb/DynamicWeb/assets/31095845/37fe518e-0a67-4a7f-b76c-7f8a738819ff)

Since you cannot create a new subfolder to Templates using the UI, but need access to the server, and the templates are global and related to the platform and not the design folder associated to an area, it would be nice if they were included in DW10:

![image](https://github.com/dynamicweb/DynamicWeb/assets/31095845/8b91bf65-2fea-4bc8-b61c-f4ec7d0fe419)

Alternatively, allow creating subfolders under Templates, so they can be uploaded using the UI.

Br.
Lars

**Comment by DynamicRas** [comment]
Hi, 

Allow me to make this request a more general one. This is a common thing for all add-ins needing templates, payment, shipping, paragraph apps etc. Best case all templates is included in the add-in and it is possible to select another from another folder or as you say, just being able to select one from the UI. 

BR Rasmus Sanggaard

**Comment by dwlhs** [comment]
Good idea to have them included in the add-ins rather than platform or Swift since they are add-in specific. Then, they are not installed on the solution, when the specific add-in is not installed.

Br.
Lars
